ui/src/app/applications/components/utils.tsx

```diff
@@ -912,16 +911,3 @@ export const BASE_COLORS = [
     '#4B0082', // purple
     '#964B00' // brown
 ];
-
-export const useActionOnLargeWindow = (action: () => void) => {
```

Comment:
Found a way to switch to CSS and avoid monitoring page size in javascript. So this function is no longer required.